core/client/src/main/java/alluxio/client/netty/NettyClient.java

```diff
@@ -106,8 +105,8 @@ public void initChannel(SocketChannel ch) throws Exception {
   private static ChannelType getChannelType() {
     if (PACKET_STREAMING_ENABLED) {
       try {
-        EpollChannelConfig.class.getField("EPOLL_MODE");
-      } catch (NoSuchFieldException e) {
+        EpollChannelOption.class.getField("EPOLL_MODE");
+      } catch (Throwable e) {
         LOG.warn("EPOLL_MODE is not supported in netty with version < 4.0.26.Final.");
```

Comment:
I am catching throwable because i am not sure about what exception it will throw if there is no  EpollChannelOption defined. NoSuchFieldException will be thrown if there is no EPOLL_MODE  field in EpollChannelOption. Just to be safe, I just catch all throwable here. But the cause of this exception should only be what is mentioned in the warning. How do you think?